tests/onnx/data/reference_scales/activation_matmul_model_mixed.json

```diff
@@ -1,7 +1,7 @@
 {
     "QuantizeLinear_X_1": {
-        "scale": 0.006937139667570591,
-        "zero_point": -11
+        "scale": 0.0007594539201818407,
+        "zero_point": 0
```

Comment:
Why it was changed?

Reply:
Because the models with updated scales have input tensors having not one tensor on 0-axis. Thus, they started to be reduced and aggregated in a new way with the consideration that 0-axis is a batch axis.

Reply:
I don't get it. Was there a bug before or you've changed the implementation for no reason?

Reply:
before for tensor with a shape - [10, 3, 4] leads to a reduced tensor - [1, 1, 1] then added to aggregator.
now [10, 3, 4] -> [10, 1, 1] then added to aggregator.
Before the reducer reduced without respect to a batch axis.

Reply:
Are you sure that `10` is the batch axis? I thought that our test models did not contain batch axes before this PR.

Reply:
This is the assumption during statistics collection

Reply:
Is that an assumption or is the fact that these models contain batch axis?
I see that `10` is an input channel, not a batch - https://github.com/openvinotoolkit/nncf/blob/develop/tests/onnx/models.py#L1245

Reply:
This is the assumption. You are right that in this particular model, this does not look like a batch

Reply:
If this is not a batch axis, then it means that your implementation is not working correctly.

Reply:
Do you have suggestions on how we can improve the implementation then?

Reply:
There are no suggestions. There is only the suspicion that these tests should not need to be updated and this case should be verified.

Reply:
The default behavior was restored to a current behavior on the develop branch. Now no scale tests were updated. 
However, there are two changes in ONNX backend scales. They might happen due the previous PRs and were not detected as the changes are less than triggered threshold.